airbyte-integrations/connectors/source-twilio/source_twilio/streams.py

```diff
@@ -457,6 +458,17 @@ def path(self, **kwargs):
         return "Services"
 
 
+class VerifyServices(TwilioStream):
+    """
+    https://www.twilio.com/docs/chat/rest/service-resource#read-multiple-service-resources
+    """
+
+    url_base = TWILIO_VERIFY_BASE_V2
```

Comment:
For some reason this endpoint seems to only accept `PageSize=100` or lower. Every other endpoint is max 1000 so it's a bit odd. I wrote in to Twilio in case it wasn't intentional, and asked if it could be documented.

```suggestion
    page_size = 100
    url_base = TWILIO_VERIFY_BASE_V2
```